### PR TITLE
Fix/events deletion

### PR DESCRIPTION
**Pull request by hoseinrafiei**
Fixed following two issues.

https://github.com/eventespresso/event-espresso-core/issues/3652
https://github.com/eventespresso/event-espresso-core/issues/3674

I tested the followings:

1- Bulk delete of multiple events using the bulk edit form.
2- Single delete of one event using the link.
3- Bulk delete when there are no events in trash.

### CODE REVIEW

File: admin_pages/events/Events_Admin_Page.core.php
Comment (tn3rb) [inline admin_pages/events/Events_Admin_Page.core.php:2217]: looks good but needs a formatting fix
(see: https://github.com/eventespresso/event-espresso-core/runs/4951951150?check_suite_focus=true)

File: core/libraries/batch/JobHandlers/PreviewEventDeletion.php
Comment (tn3rb) [inline core/libraries/batch/JobHandlers/PreviewEventDeletion.php:229]: hmmmm....

would be better if this displayed a warning/error notice that nothing was selected for deletion but that's not a quick fix, so we'll ride with this for the time being, but I'm going to create an issue for improving this UX

**Comment by hoseinrafiei** [comment]
@tn3rb Should be fine.

**Review by tn3rb — APPROVED** [review]
poifect 👌🏻

**Comment by tn3rb** [comment]
hold on Hossein, don't merge this yet, I need to fix something with the branch.

Because we use rebasing, you should normally NEVER need to merge master into your work branch. So plz don't do that in the future (and don't worry, everything's ok, rebasing is tricky but you'll get used to it)

**Review by tn3rb — CHANGES_REQUESTED** [review]
can't merge because base is messed up.. i will fix

**Review by tn3rb — APPROVED** [review]
OK Hossein, 

I've fixed this branch by resetting it back to the commit before you merged master into it, and force pushed it again. Please DO NOT pull the changes for this branch. If for whatever reason you want to work on this branch some more, you will need to delete your local copy and then re-checkout the branch again from this repo so that you get the correct updated files.

In case you didn't see the notices i posted elsewhere, here's a link to some documentation i created re: rebasing: https://github.com/eventespresso/ee-and-es-docs/discussions/81

If all of the tests on this PR have passed by the time you read this, and nobody else has merged anything to master, then you can go ahead and merge this (errr... IF you have permissions.. not sure atm) If, not then I can do so later.

**Comment by hoseinrafiei** [comment]
@tn3rb

I checked the commits logs of master branch and I didn't find any commit so I assumed that nobody merged anything into the master branch. Then I merged this one and everything works well so I had permission to do it.

BTW, Please check it and let me know if there is any issue.

**Comment by tn3rb** [comment]
hmmm... something doesn't look right with the merge...
did you select the "Squash and merge" option?

Screenshot 2022-01-26 100110

It's ok if you didn't and I'll likely leave it as is since it is only 2 commits

**Comment by hoseinrafiei** [comment]
@tn3rb 

No, unfortunately I selected the first option. I just read the rules and I saw that as the fifth rule.
I will follow from now on.

**Pull request merged**
